docs/sources/flow/reference/components/prometheus.exporter.mongodb.md

```diff
@@ -0,0 +1,121 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.exporter.mongodb
+---
+
+# prometheus.exporter.mongodb
+The `prometheus.exporter.mongodb` component embeds percona's [`mongodb_exporter`](https://github.com/percona/mongodb_exporter).
+
+{{% admonition type="note" %}}
+For this integration to work properly, you must have connect each node of your MongoDB cluster to an agent instance.
+That's because this exporter does not collect metrics from multiple nodes.
+
+You must also define two custom labels for your metrics using `relabel_configs`.
+
+The first label is `service_name`, which is how you identify this node in your cluster, for example: `ReplicaSet1-Node1`.
+
+The second label is `mongodb_cluster`, which is the name of your MongoDB cluster. It must be set to the same value for all nodes composing the cluster, for example: `prod-cluster`.
+{{% /admonition %}}
+
+We strongly recommend configuring a separate user for the Grafana Agent, giving it only the strictly mandatory security privileges necessary for monitoring your node. 
+Refer to the [Percona documentation](https://github.com/percona/mongodb_exporter#permissions) for more information.
+
+## Usage
+
+```river
+prometheus.exporter.mongodb "LABEL" {
+    mongodb_uri = "MONGODB_URI"
+}
+```
+
+## Arguments
+You can use the following arguments to configure the exporter's behavior.
+Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`mongodb_uri` | `string` | MongoDB node connection URI. | | Yes
+
+MongoDB node connection URI must be in the [`Standard Connection String Format`](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-standard-connection-string-format)
+
+
+
+## Exported fields
```

Comment:
There was a request from a customer today, to enable the use of TLS certs to connect to mongodb. It is supported on the upstream exporter, so maybe we should extrapolate those? https://raintank-corp.slack.com/archives/C01343TNHCM/p1688732477656349

Reply:
Should we do it in a follow up PR? I don't have much context of this exporter, just porting things to Flow.